inc/cachify.class.php

```diff
@@ -639,6 +639,12 @@ public static function add_dashboard_count( $items = array() )
 			)
 		);
 
+		/* Output of the cache size */
+		$cachesize = ( $size == 0 )
+			? esc_html__( 'Empty Cache', 'cachify' ) :
+			/* translators: %s: cache size */
+			sprintf( esc_html__( '%s Cache', 'cachify' ), size_format($size) );;
```

Comment:
Note that there's a duplicated semicolon here.

For improved readability, I would use a regular `if  ( empty( $size ) { $cachesize = … } else { $cachesize = … }` structure